spec/models/message_spec.rb

```diff
@@ -31,5 +31,24 @@
       expect(message).to be_valid
       expect(message.sender_name).to eq('Glowfic Constellation')
     end
+
+    it "does not notify blocking recipients" do
+      sender = create(:user)
+      recipient = create(:user)
+      create(:block, blocking_user: recipient, blocked_user: sender, block_interactions: true)
+      message = create(:message, sender: sender, recipient: recipient)
+      expect(message.visible_inbox).to eq(false)
+      expect(UserMailer).to have_queue_size_of(0)
+    end
+  end
+
+  it "validates recipient" do
+    sender = create(:user)
+    recipient = create(:user)
+    create(:block, blocking_user: recipient, blocked_user: sender, block_interactions: true)
+    message = build(:message, sender: sender, recipient: recipient)
+    expect(message).to be_valid
+    message.save!
+    expect(message.visible_inbox).to eq(false)
```

Comment:
What's happening here? This doesn't seem to be testing anything different from the test above. We also don't seem to have a test that you can't send a message to someone you've got blocked, which I would have assumed this would test.